tools/infrasys/src/keys.rs

```diff
@@ -89,9 +87,13 @@ impl KMSKeyConfigExt for KMSKeyConfig {
                     missing: "key_alias",
                 })?
             );
-            let cfn_client = CloudFormationClient::new(
-                Region::from_str(region).context(error::ParseRegionSnafu { what: region })?,
-            );
+
+            let config = aws_config::from_env()
+                .region(Region::new(region.to_owned()))
+                .load()
+                .await;
+            let cfn_client = aws_sdk_cloudformation::Client::new(&config);
```

Comment:
Nit: I'd also import `aws_sdk_cloudformation::Client as CloudFormationClient` or something so we don't need to have the full path here.